src/main/java/chronologer/command/ExportCommand.java

```diff
@@ -0,0 +1,102 @@
+import net.fortuna.ical4j.model.property.ProdId;
+import net.fortuna.ical4j.model.property.Version;
+import net.fortuna.ical4j.util.RandomUidGenerator;
+import net.fortuna.ical4j.util.UidGenerator;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.GregorianCalendar;
+
+
+/**
+ * Export the timeline as an ics file.
+ *
+ * @author Tan Yi Xiang
+ * @version v1.0
+ */
+public class ExportCommand extends Command {
+
+    private String filePath = System.getProperty("user.dir") + "/src/ChronologerDatabase/";
+    private File icsFile = new File(filePath.concat("calendar.ics"));
+
+    @Override
+    public void execute(TaskList tasks, Storage storage) throws ChronologerException {
+
+        Calendar calendar = initializeCalendar();
+        ArrayList<Task> taskList = tasks.getTasks();
+        CalendarOutputter calendarOutputter = new CalendarOutputter();
+
+        for (Task task : taskList) {
+            if (tasks.isDeadline(task)) {
+                java.util.Calendar deadlineCalendar = convertToCalendar(task.getStartDate());
+                DateTime deadlineDate = new DateTime(deadlineCalendar.getTime());
+                VEvent deadline = new VEvent(deadlineDate, task.getDescription());
+                if (task.getComment() != null) {
+                    deadline.getProperties().add(new Description(task.getComment()));
+                }
+                UidGenerator generator = new RandomUidGenerator();
+                deadline.getProperties().add(generator.generateUid());
+                calendar.getComponents().add(deadline);
+            } else if (tasks.isEvent(task) || tasks.isTodoPeriod(task)) {
+                java.util.Calendar eventStartCalendar = convertToCalendar(task.getStartDate());
+                java.util.Calendar eventEndCalendar = convertToCalendar(task.getEndDate());
+                DateTime startEventDate = new DateTime(eventStartCalendar.getTime());
+                DateTime endEventDate = new DateTime(eventEndCalendar.getTime());
+                VEvent event = new VEvent(startEventDate, endEventDate, task.getDescription());
+                if (task.getComment() != null) {
+                    event.getProperties().add(new Description(task.getComment()));
+                }
+                UidGenerator generator = new RandomUidGenerator();
+                event.getProperties().add(generator.generateUid());
+                calendar.getComponents().add(event);
+            }
+
+        }
+
+        try {
+            FileOutputStream outputStream = new FileOutputStream(icsFile);
+            calendarOutputter.output(calendar, outputStream);
+            UiTemporary.printOutput("Success,ics file written at src/ChronologerDatabase/calendar");
+        } catch (IOException e) {
+            throw new ChronologerException(ChronologerException.errorWriteCalendar());
```

Comment:
can add the UiTemporary.printOutput to print the error message as the error logger has been removed.